src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -267,6 +283,7 @@ data Parameter proposalMetadata
   | Mint MintParam
   | Transfer_contract_tokens TransferContractTokensParam
   | Token_address TokenAddressParam
+  | GetCounter (View () Nonce)
```

Comment:
Hm, apparently I misinterpreted the notion of off-chain views.

It seems to me that this entrypoint is not very temporary, in a sense that we probably shouldn't assume that everyone will be able to work with off-chain views in the nearest months. Since without counter permissions feature is unusable, it would be nice to provide some conservative mechanism. Though on the other hand, probably only one product will work with our DAOs - DAO middleware.

Okay, since we are not tied to TZIP-17 in implementing this form of getter, I'll rename it to `GetVotePermissionCounter` and mention it in spec.

Reply:
> Since without counter permissions feature is unusable

One can just read it directly from the storage without calling a view entrypoint. For off-chain access it's more convenient.